tests/cases/unittests/session.ts

```diff
@@ -0,0 +1,459 @@
+/// <reference path="..\..\..\src\harness\harness.ts" />
+
+module ts.server {
+	let lastWrittenToHost: string,
+	mockHost: ServerHost = {
+		args: [],
+		newLine: '\n',
+		useCaseSensitiveFileNames: true,
+		write: (s) => lastWrittenToHost = s,
+		readFile: () => void 0,
+		writeFile: () => void 0,
+		resolvePath: () => void 0,
+		fileExists: () => false,
+		directoryExists: () => false,
+		createDirectory: () => void 0,
+		getExecutingFilePath: () => void 0,
+		getCurrentDirectory: () => void 0,
+		readDirectory: () => void 0,
+		exit: () => void 0
+	},
+	mockLogger: Logger = {
+        close(): void {},
+        isVerbose(): boolean { return false; },
+        loggingEnabled(): boolean { return false; },
+        perftrc(s: string): void {},
+        info(s: string): void {},
+        startGroup(): void {},
+        endGroup(): void {},
+        msg(s: string, type?: string): void {},
+	};
+	
+	describe('the Session class', () => {
+		let session: Session,
+			lastSent: protocol.Message;

```

Comment:
Is it possible for us to add this one to the tslint for the future, too? It should be able to catch this.